.claude/agents/agy-pair-coder.md

```
@@ -122,22 +122,26 @@ Only escalate if BOTH fail.
 
 ### Phase 3: Verify and Signal
 1. Run tests to confirm they pass
-2. Send IMPLEMENTATION_READY message to verifier:
+2. Capture the exact revision with `git rev-parse HEAD` and send an
+   IMPLEMENTATION_READY message to verifier:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)